tasks/pangram-checker/src/main.rs

```diff
@@ -0,0 +1,84 @@
+
+#![feature(test)]
+
+extern crate test;
```

Comment:
Hm, I don't think this or `#![feature(test)` should be necessary. Can you explain your reasoning here?